mmv1/third_party/terraform/services/securitycentermanagement/resource_scc_management_organization_security_health_analytics_custom_module_test.go

```diff
@@ -0,0 +1,205 @@
+package securitycentermanagement_test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+// Custom Module tests cannot be run in parallel without running into 409 Conflict reponses.
```

Comment:
Why is this the case? Doesn't creating a custom module create an independent resource to act on?

Reply:
Yes, while each custom module is indeed an independent resource, I think creating, updating, or deleting these modules in parallel may often lead to conflict responses. This is likely due to internal rate limiting, resource contention, or transactional conflicts imposed by the API's backend infrastructure. 

BTW, this comment was present in the v1 version test case, maybe they put it as disclaimer, so I put the same for this one as well.